<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ManttoInventario</class>
 <widget class="QDialog" name="ManttoInventario">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>252</width>
    <height>285</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Mantenimiento</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="rightMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Ingreso Inventario:</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>AÃ±o:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QDateEdit" name="anio">
          <property name="displayFormat">
           <string>yyyy</string>
          </property>
          <property name="calendarPopup">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QToolButton" name="botonAgregar">
          <property name="icon">
           <iconset resource="../ContablesPrincipal.qrc">
            <normaloff>:/iconos/mas.png</normaloff>:/iconos/mas.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>16</width>
            <height>16</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QToolButton" name="botonMenos">
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="../ContablesPrincipal.qrc">
            <normaloff>:/iconos/menos.png</normaloff>:/iconos/menos.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableView" name="modeloTabla"/>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
  <action name="insertRowAction">
   <property name="text">
    <string>Insertar Fila</string>
   </property>
  </action>
  <action name="deleteRowAction">
   <property name="text">
    <string>Borrar Fila</string>
   </property>
  </action>
  <action name="modificarRowAction">
   <property name="text">
    <string>Modifica Fila</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../ContablesPrincipal.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ManttoInventario</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ManttoInventario</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
